src/lib.rs

```diff
@@ -231,3 +274,25 @@ fn rev_range_collect() {
     let x: Vec<i32> = from(14).down_to(10).into_iter().take(10).collect();
     assert_eq!(x, vec![14, 13, 12, 11, 10]);
 }
+
+#[test]
+fn as_std_range() {
+    let r = from(10).up_to(14);
+
+    let u: Vec<i32> = r.as_std_range().into_iter().take(10).collect();
+    let v: Vec<i32> = r.into_iter().take(10).collect();
+
+    assert_eq!(u, vec![10, 11, 12, 13, 14]);
+    assert_eq!(v, vec![10, 11, 12, 13, 14]);
+}
```

Comment:
these also look like good candidates for prop tests

Reply:
What's a prop test?